src/nvim/edit.c

```diff
@@ -1846,47 +1848,49 @@ static void ins_ctrl_x(void)
   }
 }
 
-/*
- * Return TRUE if the 'dict' or 'tsr' option can be used.
- */
-static int has_compl_option(int dict_opt)
+/// Check that the "dict" or "tsr" option can be used.
+///
+/// @param  dict_opt  Check "dict" when true, "tsr" when false.
+static bool has_compl_option(bool dict_opt)
```

Comment:
I wonder if it should be renamed to `check_compl_option` or something, because `has_compl_option` sounds like a pure predicate but it has side effects (error message)